routes.js

```diff
@@ -103,6 +104,11 @@ var routes = function(app) {
  */
 function sendSlackMessage(messageBody) {
   try {
+
+    // Loads environment variables from a .env file into process.env
+    // console.log("------------------------"+process.env.SLACK_WEBHOOK_URL);
+    dotenv.config();
```

Comment:
We should put it at the beginning of the file so the developer will get the acknowledgement when the script load and not only on sending msgs.